scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1524,6 +1552,20 @@ def get_sql_front50_db_conn(migrate_data_flag):
         print("Spinnaker front50 database connection established successfully")         
         return sqldb_front50
 
+def get_postgres_orca_db_conn():
+    if spin_db_type == 'postgres':
+        #Establishing the spinnaker orca postgres database connection       
+        postgresdb_orca = psycopg2.connect(database='orca', user=spin_db_username, password=spin_db_password, host=spin_db_host,port=spin_db_port)
```

Comment:
autocommit has to be enabled on this connection similar to this - platform_conn.autocommit = True

Reply:
Implemented the autocommit